packages/core/package.json

```diff
@@ -25,6 +25,9 @@
     "homepage": "https://github.com/friggframework/frigg#readme",
     "dependencies": {
         "@friggframework/assertions": "^1.0.7",
+        "@friggframework/errors": "^1.1.6",
+        "aws-sdk": "^2.1200.0",
```

Comment:
Some day, this needs to be a separate dependency for _only_ AWS flavored Frigg (so, required in by an AWS Frigg Plugin). Anywho, today's not that day, just commenting for posterity.